pkgs/development/python-modules/term-image/default.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  setuptools,
+  pillow,
+  requests,
+  urwid,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "term-image";
+  version = "0.7.2";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "AnonymouX47";
+    repo = "term-image";
+    rev = "v${version}";
```

Helps with branch/tag confusion, especially during Python mass-updates.

```suggestion
    rev = "refs/tags/v${version}";
```